streams/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/rule/condition/expression/StormSqlExpression.java

```diff
@@ -239,6 +239,14 @@ protected String getOperator(Operator operator) {
                     return " >= ";
                 case LESS_THAN_EQUALS_TO:
                     return " <= ";
+                case MULTIPLY:
```

Comment:
nit: May be safer to wrap it with space like above operators. Also applies DIVIDE, PLUS, MINUS.